### DISCUSSION TITLE
Pipenv configuration

### DISCUSSION THREAD

**Discussion by thomaspoignant** [renovatebot/renovate#10637] [Request Help]
I can't find a proper way to use renovate with pipenv.
I have a `Pipfile` and a `Pipfile.lock` in my repository but it seems that renovate don't update it.

In the renovate output I have this 
```
INFO: Dependency extraction complete (repository=serenity/serenity-module-auto-quality-check)
       "baseBranch": "master",
       "stats": {
         "managers": {
           "dockerfile": {"fileCount": 1, "depCount": 3},
           "pip_requirements": {"fileCount": 1, "depCount": 28},
           "pip_setup": {"fileCount": 1, "depCount": 1},
           "pre-commit": {"fileCount": 1, "depCount": 2}
         },
         "total": {"fileCount": 4, "depCount": 34}
       }
```

I don't know exactly what's wrong here.
Can you help me with that?

My repository configuration is :
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json"
}
```

**Comment by rarkins** [comment]
Is it located within a subdirectory?

**Reply by thomaspoignant** [reply]
No it is on top level

**Reply by thomaspoignant** [reply]
Do you have a project example that I can follow?

**Reply by rarkins** [reply]
Here was one we tested with long ago: https://github.com/renovate-tests/pipenv1


If you run with debug logging and search the logs for "pipenv" then what do you see? Maybe it can't parse the Pipfile or it thinks there's no dependencies in it
